crates/sui-types/src/base_types.rs

```diff
@@ -183,14 +185,33 @@ impl TryFrom<Vec<u8>> for SuiAddress {
     }
 }
 
-impl From<&PublicKeyBytes> for SuiAddress {
-    fn from(key: &PublicKeyBytes) -> SuiAddress {
+impl From<&BLS12381PublicKeyBytes> for SuiAddress {
```

Comment:
As discussed, we'd like to leave this to a distinct PR.

Reply:
Removed and done.